api/app/controllers/spree/api/v1/checkouts_controller.rb

```diff
@@ -0,0 +1,100 @@
+module Spree
+  module Api
+    module V1
+      class CheckoutsController < Spree::Api::BaseController
+        before_action :associate_user, only: :update
+
+        include Spree::Core::ControllerHelpers::Auth
+        include Spree::Core::ControllerHelpers::Order
+        # This before_filter comes from Spree::Core::ControllerHelpers::Order
+        skip_before_action :set_current_order
+
+        def next
+          load_order(true)
+          authorize! :update, @order, order_token
+          @order.next!
+          respond_with(@order, default_template: 'spree/api/orders/show', status: 200)
+        rescue StateMachines::InvalidTransition
+          respond_with(@order, default_template: 'spree/api/orders/could_not_transition', status: 422)
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.